conda-env/ci.yml

```diff
@@ -1,6 +1,6 @@
 # Conda continuous integration/continuous deployment (CI/CD) for running the repository's GitHub Actions CI/CD workflows.
 # End-users are not expected to use this environment.
-name: warehouse_ci
```

Comment:
Added `e3sm_` suffix to conda env yml file names and prefixs

Reply:
I'm not sure of the impact here.  It would not hurt, I suspect.

Reply:
This comment was just to make reviewers aware